projects/js-packages/components/components/product-detail/style.module.scss

```diff
@@ -0,0 +1,96 @@
+.wrapper {
+	height: 100%;
+	padding: calc( var( --spacing-base ) * 8 );
+	position: relative;
+	min-width: calc( var( --spacing-base ) * 48 );
```

Comment:
For me, we shouldn't define the `width`, the consumer should define the size and the component just fits it.

This is the normal approach for most components libraries.

Reply:
it defines the min width, easy to force to desired width. Anyway, I can remove it.

Reply:
addressed